packages/react-charting/src/utilities/utilities.ts

```diff
@@ -598,7 +598,7 @@ export function createWrapOfXLabels(wrapLabelProps: IWrapLabelProps) {
   if (node === null) {
     return;
   }
-  const axisNode = d3Select(node).call(xAxis);
+  const axisNode = d3Select(node).call(g => xAxis);
```

Comment:
does the call function have any effect here?

Reply:
We can resolve the ts errors here by also updating the node type in IWrapLabelProps and IRotateLabelProps to SVGSVGElement as done above.